icu4j/main/classes/core/src/com/ibm/icu/impl/locale/LanguageTag.java

```diff
@@ -181,7 +181,9 @@ public static LanguageTag parse(String languageTag, ParseStatus sts) {
 
         // langtag must start with either language or privateuse
         if (tag.parseLanguage(itr, sts)) {
-            tag.parseExtlangs(itr, sts);
+            // ExtLang can only be preceded by 2-3 letter language subtag.
+            if (tag._language.Length() <= 3)
```

Comment:
Just FYI - we use curly braces for if statement even the inside of if statement is one line - and this style is set in the eclipse project.

Reply:
Also - did you really build ICU4J with the change above?  it should be `tag._language.length()`, not `Length()`

Reply:
no... I haven't set up ICU4J build, yet. I was relying on bots... Thank you for catching.